src/hooks/useInstanceSettings.jsx

```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect } from 'react'
+import get from 'lodash/get'
+
+const useInstanceSettings = client => {
+  const [settings, setSettings] = useState({})
+  const [fetchStatus, setFetchStatus] = useState('idle')
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        setFetchStatus('loading')
+        const response = await client
```

Comment:
We should really create a SettingsCollection in cozy-client, no? It should be the role of a `<Query> `to handle loading / loaded

Reply:
Yes but it's one of those collections that doesn't have actual documents that can be queried etc. I think it would need special bindings in the DSL. But I can introduce a SettingsCollection *without* the Query bindings as a first step, that's a lot more straightforward?